aries_cloudagent/connections/base_manager.py

```diff
@@ -42,6 +37,7 @@ class BaseConnectionManager:
 
     RECORD_TYPE_DID_DOC = "did_doc"
     RECORD_TYPE_DID_KEY = "did_key"
+    DID_PREFIX = "did:local:{}"
```

Comment:
Finally I opted to create a default prefix for the internal DIDs. I tried multiple formulas, for instance your suggestion to reuse the old DIDDoc logic, but I found many problems to be fixed. I think that this is the cleanest way to proceed.

Reply:
Initial thoughts but this will raise a lot of red flags in the community since it looks like an unregistered DID method. We'll have to think about this one a bit more, I think.